server/data/data-library/src/main/java/org/apache/james/dlp/eventsourcing/commands/ClearCommand.java

```diff
@@ -0,0 +1,64 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.dlp.eventsourcing.commands;
+
+import java.util.Objects;
+
+import org.apache.james.core.Domain;
+import org.apache.james.eventsourcing.Command;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+
+public class ClearCommand implements Command {
+    private final Domain senderDomain;
```

Comment:
What `senderDomain` stands for in this context?

Reply:
DlpRules are defined for the domain of mail senders, what's the problem here ?

Reply:
Then I think the `private final Domain domain` in `StoreCommand` should be named `senderDomain` for the better understanding

Reply:
It's just a detail, but here you don't mind which domain you are handling... it's just a domain & the caller has to choose which one.

Reply:
ok, I'll rename to domain everywhere if you prefer that